oscillogram_classification/k_nearest_neighbors.py

```diff
@@ -0,0 +1,63 @@
+import argparse
```

Comment:
What is the reason for having a separate file for kNN clustering with all these duplicate operations instead of just adding it as another method to `cluster.py`?

Reply:
Mainly because kNN is not clustering, so it would be misleading if it is in a file called "cluster.py". Also, the test method is slightly different than in the clustering code. Many logs that are printed in "cluster.py" would not make sense when applied to kNN.